charts/hivemq-operator/templates/hivemq-operator/custom-resource.yaml

```diff
@@ -92,4 +92,6 @@ spec:
   {{- end }}
   initBusyboxImage: {{ .Values.hivemq.initBusyboxImage }}
   initDnsWaitImage: {{ .Values.hivemq.initDnsWaitImage }}
+  volumeClaimTemplates:
+  {{ .Values.hivemq.volumeClaimTemplates }}
```

Comment:
Should this be: 
{{- if .Values.hivemq.volumeClaimTemplates }}
volumeClaimTemplates: {{ toYaml .Values.hivemq.volumeClaimTemplates | nindent 4 }}
{{- end }}

Reply:
could do that, but also `.Values.hivemq.volumeClaimTemplates` has a default value of empty array, so IMO it's not necessary